Server/Plugins/APIDump/Classes/World.lua

```diff
@@ -3167,7 +3150,14 @@ function OnAllChunksAvailable()</pre> All return values from the callbacks are i
 						Type = "number",
 					},
 				},
-				Notes = "Spawns a {{cTNTEntity|primed TNT entity}} at the specified coords, with the given fuse ticks. The entity gets a random speed multiplied by the InitialVelocityCoeff, 1 being the default value.",
+				Returns =
+				{
+					{
+						Name = "EntityID",
+						Type = "number",
+					},
+				},
+				Notes = "Spawns a {{cTNTEntity|primed TNT entity}} at the specified coords, with the given fuse ticks. The entity gets a random speed multiplied by the InitialVelocityCoeff, 1 being the default value. Returns the EntityID of the spawned primed TNT entity , or -1 on failure.",
```

Comment:
Just see that the there is no check if the `Initialize` returns false.
[World.cpp#L2287](https://github.com/cuberite/cuberite/blob/74e6f41/src/World.cpp#L2287)

Will check if there are more entities / mobs that are missing a check there.